packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts

```diff
@@ -49,7 +49,7 @@ export class NoUsageData implements IUsageData, IFlowData {
   /**
    *  Noop implementation of calculatePushNormalizationFactor
    */
-  calculatePushNormalizationFactor(__events: { StackId: string; PhysicalResourceId: string; }[], __stackId: string): void {
+  calculatePushNormalizationFactor(__events: { StackId: string; PhysicalResourceId: string }[], __stackId: string): void {
```

Comment:
It has nothing to do with this PR, but this is an incredible function name. I have no idea what it does but it sounds very important 😂